.ci/pytorch/test.sh

```diff
@@ -287,25 +287,8 @@ test_perf_for_dashboard() {
   local suite="$1"
   shift
 
-  for dtype in amp float32; do
-    # Run accuracy test
-    # All the accuracy tests can be skipped once the CI accuracy checking is stable enough
-    for backend in eager aot_eager; do
```

Comment:
I removed these in https://github.com/pytorch/pytorch/pull/97543. But I don't think we have enough coverage to remove the accuracy tests for inductor yet?

Reply:
ok